webapp/src/pages/CreateChatroomPage/CreateChatroomPage.tsx

```diff
@@ -77,19 +72,10 @@ export default function CreateChatroomPage() {
     register().catch((e) => console.error(e));
   };
 
-  const chatroom: Partial<Chatroom> = {};
   return (
     <AvatarPageTemplate
       isLoading={false}
       title="create chatroom"
-      avatarProps={{
```

Comment:
I removed this, but we could use an editable avatar. The user uploads an image, and when they submit the form, we check if there is an avatar or generate it in the backend.

If we like the idea, we could create a ticket for this. It's nice to have, but not mandatory.